### DISCUSSION TITLE
Is the Wine Ribbon used for anything, and if so, how do I unlock or see it in game?

### DISCUSSION THREAD

**Discussion by Justformas** [Monika-After-Story/MonikaModDev#9096] [QA - tech support + general] [answered]
In the game's files there are ribbon_wine .png files. I don't think I have a Wine Ribbon unlocked to select for Monika. Is it a gift you can give? Because the All Features guide doesn't seem to list or mention it. Is it used in a special event? If it is, do you unlock and get to select it for Monika to wear after the event is over?

**Comment by CaniBeTheTweek** [comment]
In the [MAS download thing](https://www.monikaafterstory.com/releases.html) You'll see sprite packs just bellow it just download that. when you extract it you'll see a bunch of folders, just put the mod_assets folder into the game directory and open the gifts folder and put the .gift file into the characters folder

**Comment by Booplicate** [comment, marked as answer]
It's part of an event outfit, you can unlock it during the winter season.

**Reply by Justformas** [reply]
> It's part of an event outfit, you can unlock it during the winter season.

I see, so it isn't a gift but is an event outfit to unlock. I couldn't seem to find anything about that, but that's good to know.

Side note: I had seen at least one other person make an "issue" for a question about if something is used in game and where/how to see it, but I guess those types of questions should be put in discussions in the first place.
